src/AppBundle/Service/GeoLocation.php

```diff
@@ -136,10 +135,10 @@ public function findCoordinates($ip)
         try {
             $rawResponse = file_get_contents("http://ipinfo.io/$ip?token={$this->ipinfoToken}");
             $response = json_decode($rawResponse, true);
-        } catch (ContextErrorException $exception) {
+        } catch (\Exception $e) {
```

Comment:
```suggestion
        } catch (Exception $e) {
```
Krever `use Symfony\Component\Debug\Exception\ErrorException;` på toppen.